test/integration/_regtest.spec.ts

```diff
@@ -0,0 +1,6 @@
+import { RegtestUtils } from 'regtest-client';
+
+const APIPASS = process.env.APIPASS || 'satoshi';
+const APIURL = process.env.APIURL || 'https://regtest.bitbank.cc/1';
+
+export const regtestUtils = new RegtestUtils({ APIPASS, APIURL });
```

Comment:
The file name should not be suffixed with `.spec` since this file is not supposed to be executed by a test runner.